src/main/java/nl/ordina/jobcrawler/service/SkillService.java

```diff
@@ -94,4 +94,15 @@ public boolean delete(UUID id) {
     }
 
 
+    public void deleteReferencesToSkills() {
+        skillRepository.deleteReferencesToSkills();
+    }
+
+    public void deleteReferencesToSkill(String name) {
+        skillRepository.deleteReferencesToSkill(name);
+    }
+
+    public void deleteAll() {
+        skillRepository.deleteAll();
+    }
```

Comment:
This is for testing purposes, so it can be put in the tests

Reply:
same, is used to develop the application, so I will leave it

Reply:
is the sql file following in a future commit?